### PR TITLE
Update Cere's pallets after Substrate v3 merge

### PR DESCRIPTION
**Pull request by Raid5594**

### CODE REVIEW

File: .github/workflows/build-and-push-docker-image.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ on:
   push:
     branches:
       - dev-cere
-      - feature/CBI-2025-remove-metrics
+      - feature/CBI-2027-update-cere-pallets
```
Comment (AndreiNavoichyk) [inline .github/workflows/build-and-push-docker-image.yml:6]: Remove before merge

File: .github/workflows/run-tests-on-pull-request.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ on:
   pull_request:
     branches:
       - dev-cere
+      - feature/CBI-2027-update-cere-pallets
```
Comment (AndreiNavoichyk) [inline .github/workflows/run-tests-on-pull-request.yml]: Remove before merge

File: .gitmodules [hunk 3]
```diff
@@ -1,3 +1,4 @@
 [submodule "frame/ddc-pallet"]
 	path = frame/ddc-pallet
 	url = https://github.com/Cerebellum-Network/ddc-pallet.git
+	branch = feature/CBI-2022-update-cere-pallets
```
Comment (AndreiNavoichyk) [inline .gitmodules]: Switch to the default branch before merge

File: frame/chainbridge/src/lib.rs [hunk 4]
```diff
@@ -1,27 +1,21 @@
...
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use sp_std::marker::PhantomData;
 use frame_support::{
-	dispatch::{DispatchResult, IsSubType}, decl_module, decl_storage, decl_event, decl_error,
-	weights::{DispatchClass, ClassifyDispatch, WeighData, Weight, PaysFee, Pays, GetDispatchInfo},
-	ensure,
-	traits::{EnsureOrigin, Get},
-	Parameter,
+	decl_error, decl_event, decl_module, decl_storage,
```
Comment (AndreiNavoichyk) [inline frame/chainbridge/src/lib.rs]: Please, fix the formatting

File: frame/contracts/src/benchmarking.rs [hunk 5]
```diff
@@ -1,271 +0,0 @@
-// This file is part of Substrate.
```
Comment (AndreiNavoichyk) [inline frame/contracts/src/benchmarking.rs:1]: Should this file be removed in the branch with Substrate v3 merge?

File: frame/erc20/Cargo.toml [hunk 6]
```diff
@@ -14,19 +14,19 @@ targets = ["x86_64-unknown-linux-gnu"]
...
-frame-system = { version = "2.0.0", default-features = false, path = "../system" }
-pallet-balances = { version = "2.0.0", default-features = false, path = "../balances" }
-sp-runtime = { version = "2.0.0", default-features = false, path = "../../primitives/runtime" }
-sp-std = { version = "2.0.0", default-features = false, path = "../../primitives/std" }
-sp-io = { version = "2.0.0", default-features = false, path = "../../primitives/io" }
-sp-core = { version = "2.0.0", path = "../../primitives/core", default-features = false }
-sp-arithmetic = { version = "2.0.0", path = "../../primitives/arithmetic", default-features = false }
-pallet-chainbridge = { version = "2.0.0", default-features = false, path = "../chainbridge" }
-pallet-erc721 = { version = "2.0.0", default-features = false, path = "../erc721" }
+codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false }
```
Comment (AndreiNavoichyk) [inline frame/erc20/Cargo.toml:17]: Why not `3.0.0`?

File: frame/erc721/Cargo.toml [hunk 7]
```diff
@@ -14,18 +14,20 @@ targets = ["x86_64-unknown-linux-gnu"]
...
-codec = { package = "parity-scale-codec", version = "1.3.4", default-features = false }
-frame-support = { version = "2.0.0", default-features = false, path = "../support" }
-frame-system = { version = "2.0.0", default-features = false, path = "../system" }
-pallet-balances = { version = "2.0.0", default-features = false, path = "../balances" }
-sp-runtime = { version = "2.0.0", default-features = false, path = "../../primitives/runtime" }
-sp-std = { version = "2.0.0", default-features = false, path = "../../primitives/std" }
-sp-io = { version = "2.0.0", default-features = false, path = "../../primitives/io" }
-sp-core = { version = "2.0.0", path = "../../primitives/core", default-features = false }
-pallet-chainbridge = { version = "2.0.0", default-features = false, path = "../chainbridge" }
+codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false }
```
Comment (AndreiNavoichyk) [inline frame/erc721/Cargo.toml:17]: Why not `3.0.0`?

File: frame/erc721/Cargo.toml [hunk 8]
```diff
@@ -14,18 +14,20 @@ targets = ["x86_64-unknown-linux-gnu"]
...
+sp-std = { version = "3.0.0", default-features = false, path = "../../primitives/std" }
+sp-io = { version = "3.0.0", default-features = false, path = "../../primitives/io" }
+sp-core = { version = "3.0.0", path = "../../primitives/core", default-features = false }
+pallet-chainbridge = { version = "3.0.0", default-features = false, path = "../chainbridge" }
 
-frame-benchmarking = { version = "2.0.0", default-features = false, path = "../benchmarking", optional = true }
+frame-benchmarking = { version = "3.0.0", default-features = false, path = "../benchmarking", optional = true }
 
+[build-dependencies]
+ wasm-builder-runner = { version = "2.0.0", package = "substrate-wasm-builder-runner"}
```
Comment (AndreiNavoichyk) [inline frame/erc721/Cargo.toml:30]: Why not `3.0.0`?

File: frame/chainbridge/src/mock.rs [hunk 9]
```diff
@@ -35,16 +35,13 @@ impl frame_system::Trait for Test {
...
     type DbWeight = ();
-    type BlockExecutionWeight = ();
-    type ExtrinsicBaseWeight = ();
-    type MaximumExtrinsicWeight = ();
-    type MaximumBlockLength = MaximumBlockLength;
-    type AvailableBlockRatio = AvailableBlockRatio;
     type Version = ();
     // type ModuleToIndex = ();
-    type PalletInfo = ();
+    type PalletInfo = PalletInfo;
```
Comment (AndreiNavoichyk) [inline frame/chainbridge/src/mock.rs:41]: Previously it was `()`? Why it should be `PalletInfo` now?

File: frame/erc721/src/mock.rs [hunk 10]
```diff
@@ -21,31 +21,28 @@ parameter_types! {
...
-    type MaximumBlockWeight = MaximumBlockWeight;
     type DbWeight = ();
-    type BlockExecutionWeight = ();
-    type ExtrinsicBaseWeight = ();
-    type MaximumExtrinsicWeight = ();
-    type MaximumBlockLength = MaximumBlockLength;
-    type AvailableBlockRatio = AvailableBlockRatio;
     type Version = ();
-    type PalletInfo = ();
+    type PalletInfo = PalletInfo;
```
Comment (AndreiNavoichyk) [inline frame/erc721/src/mock.rs:41]: Previously it was `()`? Why it should be `PalletInfo` now?

File: frame/evm/Cargo.toml [hunk 11]
```diff
@@ -1,51 +0,0 @@
-[package]
```
Comment (AndreiNavoichyk) [inline frame/evm/Cargo.toml]: Should this file be removed in the branch with Substrate v3 merge?

File: frame/executive/src/lib.rs [hunk 12]
```diff
@@ -465,7 +465,7 @@ where
 		// also when running WASM.
 		frame_support::debug::RuntimeLogger::init();
 
-		<AllModules as OffchainWorker<System::BlockNumber>>::offchain_worker(*header.number());
+		<AllModules as OffchainWorker<System::BlockNumber>>::offchain_worker(*header.number())
```
Comment (AndreiNavoichyk) [inline frame/executive/src/lib.rs]: This update should be in the branch with Substrate v3 merge.

File: frame/chainbridge/src/mock.rs (hunk 9 above)
Comment (Raid5594) [reply frame/chainbridge/src/mock.rs:41]: That's the issue we were solving 2 days ago. ( ) was removed everywhere in Substrate

File: frame/evm/Cargo.toml (hunk 11 above)
Comment (Raid5594) [reply frame/evm/Cargo.toml]: The package was empty, so I removed it

File: frame/executive/src/lib.rs (hunk 12 above)
Comment (Raid5594) [reply frame/executive/src/lib.rs]: Ok

File: frame/erc721/Cargo.toml (hunk 8 above)
Comment (Raid5594) [reply frame/erc721/Cargo.toml:30]: This is the version used in the parent repo for the v3 upgrade

File: frame/erc20/Cargo.toml (hunk 6 above)
Comment (Raid5594) [reply frame/erc20/Cargo.toml:17]: There is no parity-scale-codec 3.0
The upgrade to v3.0 substrate implied upgrade from 1.3.4 to 2.0.0 for scale-codec

File: frame/contracts/src/benchmarking.rs (hunk 5 above)
Comment (Raid5594) [reply frame/contracts/src/benchmarking.rs:1]: Yes, there is no file like this is substrate v3 repo. Although this change should me probably moved as well to v3 merge

File: frame/chainbridge/src/lib.rs (hunk 4 above)
Comment (Raid5594) [reply frame/chainbridge/src/lib.rs]: dispatch::{DispatchResult, IsSubType}, decl_module, decl_storage, decl_event, decl_error,

It should be like this right?

File: .gitmodules (hunk 3 above)
Comment (Raid5594) [reply .gitmodules]: Ok

File: .github/workflows/run-tests-on-pull-request.yml (hunk 2 above)
Comment (Raid5594) [reply .github/workflows/run-tests-on-pull-request.yml]: Ok

File: frame/contracts/src/benchmarking.rs (hunk 5 above)
Comment (AndreiNavoichyk) [reply frame/contracts/src/benchmarking.rs:1]: Yes, please, move.

File: frame/chainbridge/src/lib.rs (hunk 4 above)
Comment (AndreiNavoichyk) [reply frame/chainbridge/src/lib.rs]: Yep, like in other files.

**Pull request closed without merging**